pkg/image/controller/controller.go

```diff
@@ -55,9 +217,6 @@ const retryCount = 2
 // 4. ImageStreamMapping save error
 // 5. error when marking ImageStream as imported
 func (c *ImportController) Next(stream *api.ImageStream) error {
```

Comment:
wow, I misread... you're right, there is no shared client in the image import controller... definitely need to fix that after @smarterclayton's changes go in... we'll exhaust connections with a unique transport connection pool per call to `Next()`